cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java

```diff
@@ -20,14 +22,24 @@
  * Cruise Control will use {@link HttpSession} to keep track the progress of such requests and
  * report the progress to the users.
  */
+@JsonResponseClass
 public class OperationProgress {
-  private static final String STEP = "step";
-  private static final String DESCRIPTION = "description";
-  private static final String TIME_IN_MS = "time-in-ms";
-  private static final String COMPLETION_PERCENTAGE = "completionPercentage";
+  @JsonResponseField
+  protected static final String OPERATION = "operation";
+  @JsonResponseField
+  protected static final String OPERATION_PROGRESS = "operationProgress";
   private boolean _mutable = true;
   private List<OperationStep> _steps = new ArrayList<>();
   private List<Long> _startTimes = new ArrayList<>();
+  private String _operation;
+
+  public  OperationProgress() {
```

Comment:
extra space here between `public` and `OperationProgress()`

Reply:
I think `public OperationProgress()` body can be left empty if you update the line 34 as `private String _operation = "";`.